lib/console.js

```diff
@@ -139,12 +139,14 @@ Console.prototype.dir = function dir(object, options) {
 };
 
 
-Console.prototype.time = function time(label) {
+Console.prototype.time = function time(label = 'default') {
+  label = `${label}`;
```

Comment:
Maybe we can write a line of comment to explain this? Or maybe next time who see it will be confused again.